db/memtable_list.h

```diff
@@ -128,11 +129,11 @@ class MemTableListVersion {
       LogBuffer* log_buffer);
 
   // REQUIRE: m is an immutable memtable
-  void Add(MemTable* m, autovector<MemTable*>* to_delete);
+  void Add(MemTable* m, autovector<MemTable*>* to_delete, size_t usage);
   // REQUIRE: m is an immutable memtable
   void Remove(MemTable* m, autovector<MemTable*>* to_delete);
 
-  void TrimHistory(autovector<MemTable*>* to_delete);
+  void TrimHistory(autovector<MemTable*>* to_delete, size_t usage);
```

Comment:
size of the mutable memtable. We want to use `max_write_buffer_size_to_maintain` track total size of mutable and immutable memtables.

Reply:
It may be helpful to document the meaning of `usage`. Intuitively trimming history does not require a `usage`. It is actually an optimization. A comment or better variable naming can help understand.